tests/python/test_with_dask.py

```diff
@@ -64,8 +66,8 @@ def client(cluster):
         yield dask_client
 
 
-kRows = 1000
-kCols = 10
+kRows = 300
```

Comment:
1000 is pretty trivial, right? Is it becoming a bottleneck? If the data is too small dask might put them into a single worker.

Reply:
Changed this one back. There is not too much difference.